index.html

```diff
@@ -132,8 +132,8 @@ <h1>Multi-Post Stories</h1>
                 </ul>
                 <button>See Project</button>
  
-            </div>
-            <div class="box-1">
+            </article>
+            <article class="box-1">
                 <img src="./images/ImgPlaceholder.png" alt="#">

```

Comment:
- [ ]  You should provide an alt value for your images, so that once your site is being loading with someone who has a bad connection, might no what the image was in that position.